tool.ts

```diff
@@ -218,10 +222,20 @@ export async function downloadTool(url: string, fileName?: string): Promise<stri
             if (fs.existsSync(destPath)) {
                 throw new Error("Destination file path already exists");
             }
-
-            // TODO: retries
+            
             tl.debug('downloading');
+            const  statusCodesToRetry = [502, 503, 504];
```

Comment:
Add type, extra space between type and variable, can use these for values:

502 - httpm.HttpCodes.BadGateway
503 - httpm.HttpCodes.ServiceUnavailable
504 - httpm.HttpCodes.GatewayTimeout